server/helpers/helpers.js

```diff
@@ -0,0 +1,9 @@
+const random8Password = () => {
```

Comment:
Use the crypto module from node js to generate a random password. I think we already use this for the email confirmation feature